Hearty-Vegetable-lasagana.md

```diff
@@ -1,15 +1,27 @@
 #**Hearty Vegetable Lasagna**
 
-![](http://www.feastsfromthepantry.com/wordpress/wp-content/uploads/2011/08/tumblr_lq8zjnW2T71qgmbh4.jpg)
+![](http://images.media-allrecipes.com/userphotos/560x315/620560.jpg  "Lasagana")
```

Comment:
Looks like an extra "Lasagna"

If you want to show "Lasagna" as fallback text the syntax is:

```markdown
[Lasagana](http://images.media-allrecipes.com/userphotos/560x315/620560.jpg)
```